your_daily_dose_malware/backends/malware_bazaar.py

```diff
@@ -116,7 +116,7 @@ def all_most_recent(headers_info):
 
 def scrap_malware_bazaar(hundred_recent):
     if os.getenv("MALWARE_BAZAAR_API_KEY") == "":
-        rich.print("[green] Skipping Malware Bazaar not api key")
+        rich.print("[red]Please set  MALWARE_BAZAAR_API_KEY")
         return None
     headers = {"API-KEY": os.getenv("MALWARE_BAZAAR_API_KEY")}
```

Comment:
Maybe you could do it differently, to detect for present ENV keys, and only run those backend found.
Then the system becomes dynamic.

Reply:
Ok I see I will run the check more earlier before to run any backends

Reply:
But you don't want to inform users that an API key is empty before to switch to the next? Is already what the code if the first banckend has not an API key set then the message will be printed and next will be run

Reply:
Maybe you do this with --verbose option, or with another like --check-api-keys options.
Keep in mind someone might not run or be interested in all Backends, so printing missing keys for all backends every time might be a bit annoying.

Reply:
Ok I see... About that did you consider to use typer as a CLI app builder? I will read argpase but typer is more obvious than argpase.

Reply:
Sooo?

Reply:
Sure, if you want to retry a refactor using typer, we can take a look how that looks like.

Reply:
Awesome